src/oemof/solph/processing.py

```diff
@@ -210,7 +210,7 @@ def results(model, remove_last_time_point=False):
                 columns="variable_name", values="value"
             )
             # Add empty row at the end
-            df_dict[k] = df_dict[k].append(pd.Series(), ignore_index=True)
+            df_dict[k].loc[df_dict[k].iloc[-1].name + 1, :] = float("nan")
```

Comment:
Two considerations:
1. As pandas seems to use numpy.nan internally, it might be a good idea to directly do the same.
2. You add 1 to the name of the last entry and then create the entry using that name. It might help to understand the code better if there was a comment on what "name +1" means in this context.

Reply:
1. I do not see the advantage, but if you like it better it is okay for me.
2. I shortened the expression and extended the comment.

Reply:
1. I do not see the advantage, but if you like it better I can change it.
2. I shortened the code and extended the comment. It should be clearer now.